mention-generator/src/main/java/com/brandwatch/minibcr/mentiongenerator/service/query/QueryClient.java

```diff
@@ -0,0 +1,35 @@
+package com.brandwatch.minibcr.mentiongenerator.service.query;
+
+import java.util.List;
+
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.brandwatch.minibcr.mentiongenerator.model.UserQuery;
+
+@Service
+public class QueryClient {
+
+    private static final String URL = "http://localhost:8084/query";
```

Comment:
This is ok for now - but if you dockerise these services, or were to host this anywhere, this will likely change. Potentially worth changing it to a variable now.

Reply:
Created issue/bug in project dashboard. Will fix this once query api changes gets merged. Resolving this for now.
ticket : https://github.com/Shashi-puttaswamy/mini-bcr/issues/15